torch/_torch_docs.py

```diff
@@ -8461,7 +8461,48 @@ def merge_dicts(*dicts):
 """.format(**common_args))
 
 add_docstr(torch.std, r"""
-std(input, dim, unbiased, keepdim=False, *, out=None) -> Tensor
+std(input, dim=None, *, correction=1, keepdim=False, out=None) -> Tensor
+
+Calculates the standard deviation over the dimensions specified by :attr:`dim`.
+:attr:`dim` can be a single dimension, list of dimensions, or ``None`` to
+reduce over all dimensions.
+
+The standard deviation (:math:`\sigma`) is calculated as
+
+.. math:: \sigma = \sqrt{\frac{1}{N - \delta N}\sum_{i=0}^{N-1}(x_i-\bar{x})^2}
+
+where :math:`x` is the sample set of elements, :math:`\bar{x}` is the
+sample mean, :math:`N` is the number of samples and :math:`\delta N` is
+the :attr:`correction`.
+""" + r"""
+
+{keepdim_details}
+
```

Comment:
Let's add a note here (and to the other functions about the historic behavior):

"In PyTorch 1.9 and earlier _std_ accepted an _unbiased_ argument. If _unbiased_ was True (the default) it implied `correction=1` (the current default). If _unbiased_ was False then it implied `correction=0`. The new _correction_ parameter generalizes the deprecated _unbiased_ parameter."

How does that sound?

Reply:
Tweaked the wording slightly to:
>   In PyTorch 1.8 and earlier `std` accepted an `unbiased` argument which was a less general form of the `correction` parameter. If `unbiased` was ``True`` it implied ``correction=1`` (this was the default). If `unbiased` was ``False`` then it implied ``correction=0``.